lib/tasks/openapi_generate.rake

```diff
@@ -56,11 +56,14 @@ class OpenapiGenerator
 
   def add_primary_reference_schema(inventory_collection)
     class_name                                                                 = inventory_collection.model_class.to_s
+    return if class_name.blank?
```

Comment:
I'm not sure I'm such a fan of sprinkling conditionals in these methods.  

Is there any specs on this?

Reply:
No there are no specs for this rake task. 
I think these generic conditions are better than condition just for `service_instance_tasks`.
Or what do you suggest?